app/views/welcome/index.html.erb

```diff
@@ -0,0 +1,188 @@
+<!DOCTYPE html>
```

Comment:
Is there a reason you have a whole html page here? Do you not want to render this with your base layout?